api/src/main/java/org/openmrs/module/reporting/report/service/db/HibernateReportDAO.java

```diff
@@ -225,7 +225,40 @@ public ReportRequest getReportRequestByUuid(String uuid) {
 	public void purgeReportRequest(ReportRequest request) {
 		sessionFactory.getCurrentSession().delete(request);
 	}
-	
+
+	/**
+	 * Purges the <code>ReportRequest</code> entries associated with the given report definition uuid from the database.
+	 * @param reportDefinitionUuid The report definition uuid to remove
+	 */
+	@Override
+	public void purgeReportRequests(String reportDefinitionUuid) {
+		String hql = "delete from ReportRequest r where r.reportDefinition.definition=:uuid";
+		Query query = sessionFactory.getCurrentSession().createQuery(hql);
+		query.setString("uuid", reportDefinitionUuid);
+		query.executeUpdate();
+	}
+
+
+	/**
+	 * Purges the <code>ReportDesign</code> entries associated with the given report definition uuid from the database.
```

Comment:
You do not need to repeat this. You can just use the @see annotation as here https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/api/db/hibernate/HibernateProviderDAO.java#L82

Reply:
Replaced 👍